src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -427,12 +335,10 @@ MavlinkReceiver::send_flight_information()
 {
 	mavlink_flight_information_t flight_info{};
 
-	param_t param_flight_uuid = param_find("COM_FLIGHT_UUID");
+	uint64_t flight_uuid = static_cast<uint64_t>(_p_flight_uuid.get());
```

Comment:
uint64?

Reply:
See comment below, `flight_uuid` later gets assigned to `flight_info.flight_uuid = (uint64_t)flight_uuid;` so this seemed better when I wrote it.

Reply:
Rebased on current master with updates to this section and bench tested on pixhawk 4

![image](https://user-images.githubusercontent.com/2497951/52586996-25269100-2df6-11e9-8a96-9f9f7b7be81b.png)